test/Interop/Cxx/reference/Inputs/const-ref-parameter.h

```diff
@@ -0,0 +1,22 @@
+#import <Foundation/Foundation.h>
+
+struct OptionsStruct {
+  int intOption;
+  float floatOption;
+};
+
+@interface OptionsConsumerObjC : NSObject
+
+- (nonnull instancetype)initWithOptions:(const OptionsStruct &)options;
++ (nonnull instancetype)consumerWithOptions:(const OptionsStruct &)options;
++ (int)doThingWithOptions:(const OptionsStruct &)options;
+- (float)doOtherThingWithOptions:(const OptionsStruct &)options;
+
+@end
+
+struct OptionsConsumerCxx {
+  OptionsConsumerCxx(const OptionsStruct &options);
+  static OptionsConsumerCxx build(const OptionsStruct &options);
+  static int doThing(const OptionsStruct &options);
+  float doOtherThing(const OptionsStruct &options);
```

Comment:
These method names aren't particularly helpful. Maybe you can rename them "getIntOption" and "getFloatOption" or "staticMethod" and "instanceMethod" depending on why they are interesting test cases.